commands/project/dev.ts

```diff
@@ -107,12 +107,12 @@ exports.handler = async options => {
     (!hasPublicApps && isSandbox(accountConfig));
 
   // The account that the project must exist in
-  let targetProjectAccountId = derivedAccountId;
+  let targetProjectAccountId = providedAccountId ? derivedAccountId : null;
```

Comment:
Could you update the comment to explain why we need this to be `null` when the user doesn't pass in `--account`?

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1303/commits/0ade7877391e7585c8dc0364e36257ccd348c1d1